src/v/security/acl.h

```diff
@@ -490,6 +491,36 @@ resource_pattern_filter::matches(const resource_pattern& pattern) const {
     __builtin_unreachable();
 }
 
+inline std::vector<resource_pattern>
+resource_pattern_filter::to_resource_patterns() const {
+    if (!_resource || !_name) {
+        return {};
+    }
+
+    if (
```

Comment:
dupicate logic to below no ?

Reply:
oh yeh good catch. I moved this case into the conditional below but didn't remove the original.

Reply:
oh yeh good catch. I moved this into the conditional below but didn't remove the original.